Package.swift

```diff
@@ -8,14 +8,21 @@ let package = Package(
   platforms: [.macOS(.v10_15)],
   dependencies: [
     .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.1.1"),
-    .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "0.2.0")),
-    .package(url: "https://github.com/apple/swift-tools-support-core.git", .upToNextMinor(from: "0.1.3")),
-    .package(url: "https://github.com/OpenCombine/OpenCombine.git", .upToNextMinor(from: "0.10.0")),
+    .package(
+      url: "https://github.com/apple/swift-argument-parser",
+      .upToNextMinor(from: "0.2.0")
+    ),
+    .package(
+      url: "https://github.com/apple/swift-tools-support-core.git",
+      .upToNextMinor(from: "0.1.3")
+    ),
+    .package(url: "https://github.com/OpenCombine/OpenCombine.git", from: "0.10.0"),
     .package(url: "https://github.com/vapor/vapor.git", .upToNextMinor(from: "4.5.0")),
   ],
   targets: [
-    // Targets are the basic building blocks of a package. A target can define a module or a test suite.
-    // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+    // Targets are the basic building blocks of a package. A target can define a module
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Package.swift;
  Line: 23;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 104 characters (`line_length`)